### ISSUE TITLE
[SR-13948] SPM doesn't exclude files and directories marked as exclude

### ISSUE THREAD

**Issue by swift-ci** [apple/swift-package-manager#4460]

**Comment by sivx76** [comment]
Seconded! I am the developer of a free and opensource library and I find this to be very annoying.

For Cocoapods support this didn't appear for our users. For SPM support it does. 

Is there any way to prioritize this?

Thanks

**Comment by tomerd** [comment]
@neonichu this seems to be more of an Xcode issue, correct?

**Comment by neonichu** [comment]
Yep, this is entirely about what is shown by the project navigator in Xcode.

**Comment by tomerd** [comment]
is there better way to track this then? maybe apple feedback?

**Comment by sivx76** [comment]
Opened a ticket on Feedback Assistant. Ticket ID: FB11422233 (Issue with Swift Package Manager)

@neonichu @tomerd - thank you for both of your prompt responses.

It means a lot for the opensource community. Let us know if there is any way we can help or to escalate this issue.

Benyam

**Comment by setoelkahfi** [comment]
I recently faced a similar issue. I ended up using a workaround posted [here](https://stackoverflow.com/a/70990534/1137814) to use an empty `Package.swift` file like so:

```
// swift-tools-version:5.5

import PackageDescription

let package = Package()

```

**Comment by MaxDesiatov** [comment]
@setoelkahfi is this issue reproducible for you only in Xcode or with command-line invocations of SwiftPM too?
